test/schema.spec.ts

```diff
@@ -18,7 +20,44 @@ describe.each([
   { schemaType: 'alias', schema: schemas.alias, example: alias }
 ])(`Run validate for all schemas`, ({ schemaType, schema, example }) => {
   test(`validating schema ${schemaType} should return true`, () => {
-    const isValid = validateSchema(schema, example, { snapshotEnv: 'mainnet' });
+    const isValid = validateSchema(schema, example, {
+      snapshotEnv: 'mainnet'
+    });
     expect(isValid).toBe(true);
   });
 });
+
+// Tests for turbo spaces
+
+describe.each([
+  { schemaType: 'space', schema: schemas.space, example: space },
```

Comment:
Why do you test the space validation here, as the given `space` is not a Turbo space ?

Reply:
Yes we will have turbo on space too, changed it here https://github.com/snapshot-labs/snapshot.js/pull/951/files#diff-e755a2ecce42f09829d5c7dc1de8853d1d00ef56eaadc2709601c87b9be8ddfb So i thought better to add a placeholder